working-groups.md

```diff
@@ -0,0 +1,63 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Calls: [Weekly][calendar]  
+👪 Members: [**@business-planning-wg**][t-biz]  
+✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  
+
+### [Finance][fin-wg]
+
+WG for stewarding the financial sustainability of the organization, this includes establishing accounting practices, maintaining financial accounts, processing invoices, expenses, and payroll, and creating reports on our financial statements and tax obligations.
+
+📅 Calls: [Weekly][calendar]   
+👪 Members: [**@finance-wg**][t-fin]  
+✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  
+
+### [Governance][gov-wg]
+
+WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).
+
+📅 Calls: [Biweekly][calendar]  
```

Comment:
Looks pretty weekly to me :D

Reply:
I think we're weekly for this three weeks, but when I'm looking at the calendar ahead it moves back to biweekly? (We can confirm on Wednesday -- but that seems like the right rhythm for governance)